Dockerfile

```diff
@@ -32,6 +32,11 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
     autoconf \
     libglpk-dev
 
+# AWS
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.0.30.zip" -o "awscliv2.zip" && \
```

Comment:
I assume that the OS dependencies are installed ([ref](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html))?

> The AWS CLI version 2 uses `glibc`, `groff`, and `less`. These are included by default in most major distributions of Linux.

Reply:
I did test it... `groff` is actually not installed, but doesn't seem to matter except for help commands. I could add it though.

Reply:
Might as well?

Reply:
I added `groff` and `less` but I think `glibc` is part of the core? Things work without adding it, so I am assuming until an error comes that it is fine (Also, the `apt` packages seem to only be `glibc` docs and source, which we should not need?)